src/Adapter/Order/OrderAmountUpdater.php

```diff
@@ -292,6 +294,13 @@ private function getProductFromCart(array $cartProducts, int $productId, int $pr
 
             return $productMatch && $combinationMatch ? $item : null;
         });
+
+        // This shouldn't happen, if it does something was not done before updating the Order (removing an OrderDetail maybe)
+        if (null === $cartProduct) {
+            throw new OrderException('Could not find the product in cart, meaning Order and Cart are not synced');
```

Comment:
no translation needed for this ?

Reply:
Not in error messages This one is not destined to be displayed on front